posthog/session_recordings/realtime_snapshots.py

```diff
@@ -85,8 +85,10 @@ def get_realtime_snapshots(team_id: str, session_id: str, attempt_count=0) -> Op
             snapshots = []
 
             for s in encoded_snapshots:
+                # s[0] is the content
+                # s[1] is the time the content was written to redis
                 for line in s[0].splitlines():
-                    snapshots.append(json.loads(line))
+                    snapshots.append(line.decode("utf8"))
```

Comment:
we were json loading only to then json dump to build the response

so let's not... calling `.decode` to convert from byte string to string which might be avoidable to...